File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L2/04/method.ts
import * as readline from 'readline';

class Method {
  // Main program
  public static main(args: string[]): void {
    const rl = readline.createInterface({
      input: process.stdin,
      output: process.stdout
    });

    rl.question('Enter an integer: ', (input) => {
      const n = parseInt(input);
      // Call method
      this.beautyReverse(n);
      rl.close();
    });
  }

  // Reverse method
  public static beautyReverse(num: number): void {
    while (num !== 0) {
      const r = num % 10;
      process.stdout.write(r.toString());
      num = Math.floor(num / 10);
    }
    console.log();
  }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L2/04/method.py
import sys

def main():
    print("Enter an integer: ", end="")
    n = int(input())
    beauty_reverse(n)

def beauty_reverse(num):
    while num != 0:
        r = num % 10
        print(r, end="")
        num //= 10
    print()

if __name__ == "__main__":
    main()

